docs/man/borg-init.1

```diff
@@ -81,6 +76,14 @@ a different keyboard layout.
 You can change your passphrase for existing repos at any time, it won\(aqt affect
 the encryption/decryption key or other secrets.
 .SS Encryption modes
+.sp
+For each repo, the user may select the degree of encryption, and the Hash/MAC 
+algorithm. The table below shows what string to enter, to obtain each combination
+of encryption degree and Hash/MAC algorithm.
```

Comment:
usually one would say encryption algorithm.

sadly, in borg's case this is currently rather "enc. algo plus where the key is stored".

Reply:
Yes, my mistake, "degree of encryption" is not the correct terminology. I wanted to make it clear that the user has two choices: 
1. none vs authentication vs repokey vs keyfile
2. SHA-256 vs blake2b

How about:
For each repo, the user may choose between four encryption options:
1. none
2. authentication
3. repokey (store key with the repo, only passphrase needed for access)
4. keyfile (store key in $HOME/.config/borg/keys, passphrase and key needed for access)
If encryption is used, the user can select the algorithm from
1. SHA-256
2. Blake2b
The strings to select combinations of each of these options are shown in the following table.

Reply:
AES-CTR is the encryption algorithm borg currently uses.
HMAC-SHA256 or BLAKE2B is the authentication algorithm used.

Reply:
Ok, I'll try and include that information too:
When initiating each repo, the user has several options which are indicated by inputs to the `--encryption` argument. Firstly, the user may choose to have no encryption, authentication only, or to authentication and encryption. If the user chooses to encrypt the repository, they can choose where to store the key:
1. repokey (store key with the repo, only passphrase needed for access)
2. keyfile (store key in $HOME/.config/borg/keys, passphrase and key needed for access)
If the user chooses to use authentication (including authentication and encryption), they can choose between two different authentication algorithms:
1. SHA-256
2. Blake2b
The strings to select combinations of each of these options are shown in the following table.

Reply:
The key is also needed for the "authenticated" mode.

wording in 1 is misleading. you always need to have key AND passphrase to access your data.

but if you store the key in the repo, a potential attacker with repo access already has it, so the security offered by this mode is equivalent to "passphrase only" (because this is what the attacker does not have yet).

Authentication is HMAC-SHA-256 (not SHA-256).